Description: Use default settings for MAAS txlongpoll
 Use default settings for MAAS txlongpoll. These default settings
 include  oops directory, Message broker configuration, logfile.
Author: Andres Rodriguez <andreserl@ubuntu.com>

Index: maas-0.1+bzr398+dfsg.orig/etc/txlongpoll.yaml
===================================================================
--- maas-0.1+bzr398+dfsg.orig.orig/etc/txlongpoll.yaml	2012-04-03 09:49:48.109427702 -0400
+++ maas-0.1+bzr398+dfsg.orig/etc/txlongpoll.yaml	2012-04-03 09:53:07.276282800 -0400
@@ -17,7 +17,7 @@
   # or directories other than what the oops machinery creates there.
   #
   # directory: ""
-  directory: "logs/oops"
+  directory: "/var/log/maas/oops"
   ## The reporter used when generating OOPS reports.
   # reporter: "LONGPOLL"
   reporter: "maas-txlongpoll"
@@ -25,14 +25,14 @@
 ## Message broker configuration.
 #
 broker:
-  # host: "localhost"
-  # port: 5672
-  # username: "guest"
-  # password: "guest"
-  # vhost: "/"
+  host: "localhost"
+  port: 5672
+  username: "maas_longpoll"
+  password: "maaslongpoll"
+  vhost: "/maas_longpoll"
 
 ## Where to log. This log can be rotated by sending SIGUSR1 to the
 ## running server.
 #
 # logfile: "txlongpoll.log"
-logfile: "logs/txlongpoll.log"
+logfile: "/var/log/maas/txlongpoll.log"
